configure.ac: fork() during registry scan is unsafe on Cygwin (bug 555978).

--- origsrc/gstreamer-0.10.22/configure.ac	2009-01-19 14:35:32.000000000 -0600
+++ src/gstreamer-0.10.22/configure.ac	2009-02-28 23:44:21.562500000 -0600
@@ -225,7 +225,10 @@ case $host_os in
 	AC_DEFINE_UNQUOTED(GST_EXTRA_MODULE_SUFFIX, [".dylib"], [Extra platform specific plugin suffix])
         AC_DEFINE_UNQUOTED(GST_HAVE_UNSAFE_FORK, 1, [Defined when registry scanning through fork is unsafe])
 	;;
-     cygwin* | mingw* | msvc* | mks*)
+     cygwin*)
+	AC_DEFINE_UNQUOTED(GST_HAVE_UNSAFE_FORK, 1, [Defined when registry scanning through fork is unsafe])
+	;;
+     mingw* | msvc* | mks*)
         dnl HAVE_WIN32 currently means "disable POSIXisms".
         AC_DEFINE_UNQUOTED(HAVE_WIN32, 1, [Defined if compiling for Windows])
         ;;
